msteams-platform/concepts/apps/apps-upload.md

```diff
@@ -1,56 +1,55 @@
-
-For bots designed only for 1:1 contexts, see [Adding a bot for 1:1 chat only](~/concepts/bots/bots-test#adding-a-bot-for-11-chat-only) for an alternate way to access for testing purposes.
+To test your app experience within Microsoft Teams, you need to upload your app to Teams. uploading adds the app to the team you select, and you and your team members can interact with it like end users.
 
 > [!NOTE]
-> Sideloading an updated package for an existing app with a bot might not show tab changes when viewed through the Conversations window. We recommend you access via the Apps fly-out, or test on a clean test environment.
+> Uploading an updated package for an existing app with a bot might not show tab changes when viewed through the Conversations window. It's better to access it via the Apps fly-out, or test on a clean test environment.
+
+For bots designed only for 1:1 contexts, see [Adding a bot for 1:1 chat only](~/concepts/bots/bots-test#adding-a-bot-for-11-chat-only) for an alternate way to access for testing purposes.
 
-## Create your sideload package
+## Create your upload package
 
-For both development and final Office Store submission, you must create a sideloadable package that contains the information to describe your experience. The package, a .zip file, contains the application manifest and icons that uniquely define your experience.
+For development as well as Office Store submission you must create an uploadable package that contains the information to describe your experience. The package, a .zip file, contains the application manifest and icons that uniquely define your experience.
 
-To create your sideload package, see [Create the package for your Microsoft Teams app](~/publishing/apps-package).
+To create a upload package, see [Create the package for your Microsoft Teams app](~/publishing/apps-package).
 
 ## Load your package into a team
 
-With your package created, you can now load it into a team of your choosing. This adds the experience as an available integration for all users in the selected team.
+With your package created, you can now upload it into a team. Once uploaded it will be available for all users in the selected team, and only the users of that team.
 
 > [!NOTE]
-> For sideloading to work, your tenant admin must first [enable sideloading of apps](/microsoftteams/admin-settings).
+> For uploading to work, your tenant admin must first [enable uploading of apps](/microsoftteams/admin-settings).
 
-1. Create a new team for testing, if necessary. Click **Create team** at the bottom of the left-hand panel.
+1. Create a new team for testing, if necessary. Click **Create and join team** at the bottom of the left-hand panel.
 
-2. In the target team, choose **More options** (**&#8943;**) and choose **View team**.
+2. In the target team, choose **More options** (**&#8943;**) and choose **Manage team**.
 
-   ![View team](~/assets/images/tab_view_team.png)
+   ![View team](~/assets/images/ManageTeam.png)
```

Comment:
I think the convention is to have lowercase filenames. Not important to change though.